### PR TITLE
add deploy-9 docs

### PR DESCRIPTION
**Pull request by gomathyk**
Encryption section

### CODE REVIEW

File: en/docs/deploy/security/configure-keystores-in-wso2-products.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+After you have [created a new keystore](../../../deploy/security/create-new-keystores) and updated the `client-truststore.jks` file, you need to update a few configuration files in order to make the keystores work. Note that keystores are used for multiple functions in WSO2 Identity Server, which includes authenticating communication over SSL/TLS, encrypting pass words, and other confidential information in configuration files etc. Therefore, you must update the specific configuration files with the updated keystore information. For example, you may have separate keystores for the purpose of encrypting
```
Comment (dewniMW) [inline en/docs/deploy/security/configure-keystores-in-wso2-products.md:3]:
```suggestion
After you have [created a new keystore](../../../deploy/security/create-new-keystores) and updated the `client-truststore.jks` file, you need to update a few configuration files in order to make the keystores work. Note that keystores are used for multiple functions in WSO2 Identity Server, which includes authenticating communication over SSL/TLS, encrypting passwords, and other confidential information in configuration files, etc. Therefore, you must update the specific configuration files with the updated keystore information. For example, you may have separate keystores for the purpose of encrypting
```

File: en/docs/deploy/security/configure-keystores-in-wso2-products.md [hunk 2]
```diff
@@ -19,20 +19,20 @@ The `wso2carbon.jks` keystore file, which is shipped with WSO2 Identity Server,
...
+Currently, the primary keystore configured in `deployment.toml` is used for internal data encryption (encrypting data in internal datastores and configuration files) as well as for signing messages that are communicated with external parties. However, it is sometimes a common requirement to have separate keystores for communicating messages with external parties (such SAML, OIDC id_token signing) and for encrypting information in internal datastores. This is because, for the first scenario of signing messages, the keystore certificates need to be frequently renewed. However, for encrypting information in internal datastores, the keystore certificates should not be changed frequently because the data that is already encrypted will become unusable every time the certificate changes.
```
Comment (dewniMW) [inline en/docs/deploy/security/configure-keystores-in-wso2-products.md:33]: `deployment.toml` -> shouldn't we provide the directory path for this file?

File: en/docs/deploy/security/create-new-keystores.md [hunk 3]
```diff
@@ -15,20 +15,20 @@ Let's get started with creating new keystores.
...
 
 ### Create a keystore using an existing certificate
 
-Secure Sockets Layer (SSL) is a protocol that is used to secure communication between systems. This protocol uses a public key, a private key and a random symmetric key to encrypt data. As SSL is widely
+Secure Sockets Layer (SSL) is a protocol that is used to secure communication between systems. This protocol uses a public key, a private key, and a random symmetric key to encrypt data. As SSL is widely
 used in many systems, certificates may already exist that can be reused. In such situations, you can use an already existing CA-signed certificate to generate your keystore for SSL by using OpenSSL and Java
 keytool.
 
-1.  First, you must export certificates to the **[PKCS12/PFX](http://en.wikipedia.org/wiki/PKCS_12)** format. 
+1.  Export certificates to the **[PKCS12/PFX** format.
```
Comment (dewniMW) [inline en/docs/deploy/security/create-new-keystores.md:31]: why was this link removed?

File: en/docs/deploy/security/create-new-keystores.md [hunk 4]
```diff
@@ -168,18 +168,14 @@ Follow the steps given below to import the same CA-signed public key certificate
     ```
 
     !!! note
-        'wso2carbon' is the keystore password of the default `client-truststore.jks` file.
+        `wso2carbon` is the keystore password of the default `client-truststore.jks` file.
     
 
-    Now, you have an SSL certificate stored in a Java keystore and a public key added to the `client-truststore.jks` file. Note that both these files should be in the `<IS_HOME>/repository/resources/security/` directory. You can now replace the default `wso2carbon.jks` keystore in your product with the newly created keystore by updating the relevant configuration files in your product.
+    Now, you have an SSL certificate stored in a Java keystore and a public key added to the `client-truststore.jks` file. Note that both these files should be in the `<IS_HOME>/repository/resources/security/` directory. You can now replace the default `wso2carbon.jks` keystore in your WSO@ Identity Server instance with the newly created keystore by updating the relevant configuration files.
```
Comment (dewniMW) [inline en/docs/deploy/security/create-new-keystores.md:174]:
```suggestion
    Now, you have an SSL certificate stored in a Java keystore and a public key added to the `client-truststore.jks` file. Note that both these files should be in the `<IS_HOME>/repository/resources/security/` directory. You can now replace the default `wso2carbon.jks` keystore in your WSO2 Identity Server instance with the newly created keystore by updating the relevant configuration files.
```

File: en/docs/deploy/security/manage-keystores-with-the-ui.md [hunk 5]
```diff
@@ -17,11 +17,11 @@ WSO2 Identity Server Management Console enables viewing keystore details such as
 
 ## Upload keystores  
 
-Follow the instructions below to upload a keystore file to via the Management Console.
+Follow the instructions below to upload a keystore file via the WSO2 Identity Server Management Console.
 
 <ol>
     <li>
-        <p>In the <b>Main</b> menu of the <a href="../../../deploy/get-started/get-started-with-the-management-console/">Management Console</a>, click <b>Manage > Keystores > Add</b>.</p>
+        <p>In the <b>Main</b> menu of the WSO2 Identity Server Management Console (<code>https://&lt;IS_HOST&gt;:&lt;PORT&gt;/carbon</code>), click <b>Manage > Keystores > Add</b>.</p>
```
Comment (dewniMW) [inline en/docs/deploy/security/manage-keystores-with-the-ui.md:24]: in some docs, we have not bolded `>`. Better if we can decide on a common format for this

File: en/docs/deploy/security/manage-keystores-with-the-ui.md [hunk 6]
```diff
@@ -63,12 +63,12 @@ Follow the instructions below to upload a keystore file to via the Management Co
 </ol>
 
 
-## Viewing keystores 
+## View keystores
```
Comment (dewniMW) [inline en/docs/deploy/security/manage-keystores-with-the-ui.md:66]: add a line separator before h2

File: en/docs/deploy/security/manage-keystores-with-the-ui.md [hunk 7]
```diff
@@ -63,12 +63,12 @@ Follow the instructions below to upload a keystore file to via the Management Co
...
 
 
-## Viewing keystores 
+## View keystores 
 
 Follow the instructions below to view the details of the keystore that you uploaded. 
 
 <ol>
-    <li>In the <b>Main</b> menu of the Management Console, click <b>Manage > Keystores > List</b>. The <b>Key Store List</b> page appears.</li>
+    <li>In the <b>Main</b> menu of the WSO2 Identity Server Management Console, click <b>Manage > Keystores > List</b>. The <b>Key Store List</b> page appears.</li>
```
Comment (dewniMW) [inline en/docs/deploy/security/manage-keystores-with-the-ui.md:71]: https://github.com/wso2/docs-is/pull/2649/files#r646430811

File: en/docs/deploy/security/use-symmetric-encryption.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 # Configurations Related to Symmetric Key Encryption
 
-This section explains the configurations related to [symmetric key encryption](../../../deploy/security/symmetric-overview). From WSO2 Identity Server 5.11.0 onwards, symmetric encryption will be used by default. You can switch to [asymmetric key encryption](../../../deploy/security/use-asymmetric-encryption) if required.
+This section explains the configurations related to [symmetric key encryption](../../../deploy/security/symmetric-overview). This version of WSO2 Identity Server uses symmetric encryption by default. You can switch to [asymmetric key encryption](../../../deploy/security/use-asymmetric-encryption) if required.
```
Comment (dewniMW) [inline en/docs/deploy/security/use-symmetric-encryption.md:3]: `This version of WSO2 Identity Server uses symmetric encryption by default.` -> contradicting with https://github.com/wso2/docs-is/pull/2649/files#diff-87b30ea095c7c836e6e9202cfaba9d93ed0bf7545260261857f13dfbe0705d17R5

File: en/docs/deploy/security/configure-keystores-in-wso2-products.md (hunk 2 above)
Comment (gomathyk) [reply en/docs/deploy/security/configure-keystores-in-wso2-products.md:33]: We have given it in the steps below.

File: en/docs/deploy/security/create-new-keystores.md (hunk 3 above)
Comment (gomathyk) [reply en/docs/deploy/security/create-new-keystores.md:31]: We do not point to wikipedia information as a practice due to chances of inaccuracy.

File: en/docs/deploy/security/manage-keystores-with-the-ui.md (hunk 5 above)
Comment (gomathyk) [reply en/docs/deploy/security/manage-keystores-with-the-ui.md:24]: We can do a bulk replace and send a separate PR for this. Personally, I don't think it is a major inconsistency.

Comment (dewniMW) [reply en/docs/deploy/security/manage-keystores-with-the-ui.md:24]: IMO this is not something which we can achieve easily with a bulk replace

Comment (gomathyk) [reply en/docs/deploy/security/manage-keystores-with-the-ui.md:24]: As discussed, can do it with a few patterns. Will keep a note.

**Review by dewniMW — APPROVED** [review]

**Pull request merged**